model/src/pyrenew/model/rtinfectionsrenewalmodel.py

```diff
@@ -207,13 +207,14 @@ def sample_infections_obs(
         name: str | None = None,
         **kwargs,
     ) -> tuple:
-        """Sample number of hospitalizations
+        """
+        Sample number of hospital admissions
```

Comment:
This function samples observed infections, not observed admissions. Docstring needs to be updated to reflect that.

```suggestion
        Sample observed infections according 
        to an observation process, if one has 
        been specified.
```